modules/indexing/src/test/java/org/apache/ignite/internal/processors/cache/persistence/db/IgniteCacheGroupsWithRestartsTest.java

```diff
@@ -348,12 +347,14 @@ private VisorFindAndDeleteGarbageInPersistenceTaskResult executeTask(
         IgniteEx ignite,
         boolean deleteFoundGarbage
     ) {
-        VisorFindAndDeleteGarbageInPersistenceTaskArg group = new VisorFindAndDeleteGarbageInPersistenceTaskArg(
-            Collections.singleton(GROUP), deleteFoundGarbage, null);
+        CacheFindGarbageCommandArg arg0 = new CacheFindGarbageCommandArg();
+
+        arg0.groups(new String[] {GROUP});
+        arg0.delete(deleteFoundGarbage);
 
         UUID id = ignite.localNode().id();
 
-        VisorTaskArgument arg = new VisorTaskArgument(id, group, true);
+        VisorTaskArgument arg = new VisorTaskArgument(id, arg0, true);
```

Comment:
Let's move the construction of `VisorTaskArgument` to `execute(...)`, shall we?
So it is possible to remove `arg0` to `arg`